tools/gcp_cli.py

```diff
@@ -347,6 +351,7 @@ def InstanceNetworkQuarantine(args: 'argparse.Namespace') -> None:
   forensics.InstanceNetworkQuarantine(args.project,
       args.instance_name, exempted_ips, args.enable_logging )
 
+
```

Comment:
Extra blank line

Reply:
The linter (at least in my VSCode) likes 2 blank lines between each "high level def"

Reply:
Ugh, my bad, you're right. Didn't remember this until I started writing python again and PyCharm screamed at me. Ignore!